Libraries/Lists/VirtualizedList.js

```diff
@@ -2161,18 +2187,23 @@ class CellRenderer extends React.Component<
       ? [styles.row, inversionStyle]
       : inversionStyle;
     const result = !CellRendererComponent ? (
-      /* $FlowFixMe[incompatible-type-arg] (>=0.89.0 site=react_native_fb) *
+      <View
+        style={cellStyle}
+        onLayout={onLayout}
+        onFocusCapture={onFocusCapture}
```

Comment:
It would probably be better to attach this higher in the list, then to determine cell based on event target.